dashboards/aws-apprunner/aws-apprunner.json

```diff
@@ -0,0 +1,301 @@
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT count(`aws.apprunner.ActiveInstances`) FROM Metric WHERE `collector.name` = 'cloudwatch-metric-streams' FACET `aws.accountId` SINCE 24 HOURS AGO TIMESERIES"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "CPU Utilization",
+          "layout": {
+            "column": 5,
+            "row": 4,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT average(`aws.apprunner.CPUUtilization`) FROM Metric WHERE `collector.name` = 'cloudwatch-metric-streams' FACET `aws.accountId` SINCE 24 HOURS AGO TIMESERIES"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Memory Utilization",
+          "layout": {
+            "column": 9,
+            "row": 4,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT average(`aws.apprunner.MemoryUtilization`) FROM Metric WHERE `collector.name` = 'cloudwatch-metric-streams' FACET `aws.accountId` SINCE 24 HOURS AGO TIMESERIES"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Application logs ",
+          "layout": {
+            "column": 1,
+            "row": 7,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT count(*) FROM Log WHERE `aws.logGroup` LIKE '%/application' SINCE 1648578495571 UNTIL 1648664890571 facet `aws.logGroup` TIMESERIES MAX"
```

Comment:
```suggestion
                "query": "SELECT count(*) FROM Log WHERE `aws.logGroup` LIKE '%/application' SINCE 30 minutes ago facet `aws.logGroup` TIMESERIES MAX"
```

Reply:
I would not have this query as a hardcoded time, so you can pick some relative time and go with it.